packages/grafana-ui/src/components/Tags/Tag.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
+import { getTagColorsFromName } from '../../utils';
+
+interface TagProps {
+  /** Name of the tag to display */
+  name: string;
+  onClick?: (name: string) => any;
+  /** Custom styles for the wrapper component */
+  className?: string;
+  [key: string]: any;
+}
+
+export const Tag: FC<TagProps> = ({ name, onClick, className, ...rest }) => {
+  const theme = useTheme();
+  const styles = getTagStyles(theme, name);
+
+  const onTagClick = () => {
+    typeof onClick === 'function' && onClick(name);
+  };
+
+  return (
+    <span key={name} onClick={onTagClick} className={cx(styles.wrapper, className)} {...rest}>
```

Comment:
The tag styles atm are quite specific to the Search functionality, so there's a high probability they might need to be customised (like it's done from the `TagList`). Not 100% sure if this is the preferred way to do this in emotion.